### PR TITLE
Update: Glodon.CADReader version 3.5.2.15

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/116569&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/g/Glodon/CADReader/3.5.2.15/Glodon.CADReader.installer.yaml [hunk 1]
```diff
@@ -9,14 +9,13 @@ UpgradeBehavior: install
...
 - dxf
-ProductCode: '{F820B3E7-90F4-4420-8C49-3B0107740927}_is1'
-ReleaseDate: 2023-06-06
 AppsAndFeaturesEntries:
 - DisplayName: CADReader v3.5.2.15
   ProductCode: '{F820B3E7-90F4-4420-8C49-3B0107740927}_is1'
 Installers:
 - Architecture: x86
-  InstallerUrl: https://yuntu-f.glodon.com/app/CADReaderIntl.exe
+  InstallerUrl: https://yuntu-f.kuaicad.com/app/CADReaderIntl_3.5.2.15.exe
```
[the remark this answers is not in the document]
Comment (SpecterShell) [reply manifests/g/Glodon/CADReader/3.5.2.15/Glodon.CADReader.installer.yaml:17]: https://cadreader.glodon.com/query/update/cadpc?clientVersion=3.4.3.12&cadpcClientBits=32

[the remark this answers is not in the document]
Comment (stephengillie) [reply manifests/g/Glodon/CADReader/3.5.2.15/Glodon.CADReader.installer.yaml:17]: Hi @SpecterShell,

Similar to https://github.com/microsoft/winget-pkgs/pull/116571/files#r1296182203, can we replace the `PackageUrl` with the update API link you provided above? And again, I'd suggest this but GitHub won't allow this for some reason.

[the remark this answers is not in the document]
Comment (SpecterShell) [reply manifests/g/Glodon/CADReader/3.5.2.15/Glodon.CADReader.installer.yaml:17]: @stephengillie Sorry for late reply,

What do you mean by replacing the `PackageUrl` with the API link?

The `InstallerUrl` has already been replaced by the link provided by the API and this PR is intended to do so.

The API link returns a JSON text containing the link, not the real installer that link points to, so I don't think replacing with the API link would work. 🤔

[the remark this answers is not in the document]
Comment (stephengillie) [reply manifests/g/Glodon/CADReader/3.5.2.15/Glodon.CADReader.installer.yaml:17]: I discussed with the team and this is fine. Sorry for causing concern.

**Review by Trenly — APPROVED** [review]

**Pull request merged**